module2/projects/little_shop/dry_exception_handling.md

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
Should this be in this lesson? Or should this be a stand-alone resource? I'm thinking this could be a stand alone and could be linked to the error-handling class as a Next Step or something. It might be too easily missed if this lives in the project.

Reply:
Happy to move to a separate lesson page!